src/allmydata/test/cli/wormholetesting.py

```diff
@@ -61,23 +60,23 @@ class MemoryWormholeServer(object):
     :ivar _waiters: Observers waiting for a wormhole to be created for a
         specific application id and relay URL combination.
     """
-    _apps: dict[ApplicationKey, _WormholeApp] = field(default=Factory(dict))
-    _waiters: dict[ApplicationKey, Deferred] = field(default=Factory(dict))
+    _apps: Dict[ApplicationKey, _WormholeApp] = field(default=Factory(dict))
```

Comment:
My understanding is that with `from __future__ import annotations`, you don't need `typing.Dict`, you can just use `dict`. So this change is unnecessary, I think.

Reply:
Changed back to dict. Thank you for your feedback!